cmd/internal/ec2/postman-insights-agent.tmpl

```diff
@@ -1,3 +1,10 @@
+# POSTMAN_ENV is optional. If left blank, the agent will send witness data
+# to default Postman environment
+# For example: 
+#   POSTMAN_ENV=STAGE
+#
+POSTMAN_ENV={{.PostmanEnv}}
+
```

Comment:
Normal users should not have to worry about this.  This is functionality only for our own testing and dogfooding.

So, it should certainly not appear first in the file, above everything else.  I think it would be better if it were entirely absent when POSTMAN_ENV is not in use.  (The flip side is, it doesn't need to be nicely documented like the other options.)